src/common/type_utils.cpp

```diff
@@ -70,6 +70,8 @@ std::string TypeUtils::entryToString(const LogicalType& dataType, const uint8_t*
         return TypeUtils::toString(*reinterpret_cast<const timestamp_t*>(value));
     case LogicalTypeID::INTERVAL:
         return TypeUtils::toString(*reinterpret_cast<const interval_t*>(value));
+    case LogicalTypeID::BLOB:
```

Comment:
I also see `RECURSIVE_REL` `ANY` and `POINTER` not covered here.
Should `RECURSIVE_REL` be also added here?